contracts/interfaces/IAMPLSupplyPolicy.sol

```diff
@@ -0,0 +1,5 @@
+pragma solidity 0.4.24;
```

Comment:
Similar to IAMPLGovernance, don't think we need this.

Oh wait, is this for the sattelite chains?

Reply:
No not for satellite chains